--- inkscape-0.41/src/extension/internal/gnome.cpp.orig	2004-12-28 18:41:54.000000000 +0100
+++ inkscape-0.41/src/extension/internal/gnome.cpp	2005-02-14 02:46:48.614262360 +0100
@@ -392,6 +392,12 @@
 	return;
 }
 
+unsigned int PrintGNOME::comment(Inkscape::Extension::Print *mod, char const *comment) {
+    return 0;
+}
+
+
+
 }  /* namespace Internal */
 }  /* namespace Extension */
 }  /* namespace Inkscape */
