http://bazaar.launchpad.net/~inkscape.dev/inkscape/trunk/revision/10398

=== modified file 'src/graphlayout.cpp'
--- src/graphlayout.cpp	2010-12-12 08:40:34 +0000
+++ src/graphlayout.cpp	2011-07-02 12:08:37 +0000
@@ -156,11 +156,12 @@
          ++i)
     {
         SPItem *iu=*i;
-        map<string,unsigned>::iterator i=nodelookup.find(iu->getId());
-        if(i==nodelookup.end()) {
+        map<string,unsigned>::iterator i_iter=nodelookup.find(iu->getId());
+        map<string,unsigned>::iterator i_iter_end=nodelookup.end();
+        if(i_iter==i_iter_end) {
             continue;
         }
-        unsigned u=i->second;
+        unsigned u=i_iter->second;
         GSList *nlist=iu->avoidRef->getAttachedConnectors(Avoid::runningFrom);
         list<SPItem *> connectors;
 

