diff -ruN inkscape-0.44.1./configure.ac inkscape-0.44.1/configure.ac
--- inkscape-0.44.1./configure.ac	2006-09-07 09:19:48.000000000 +0200
+++ inkscape-0.44.1/configure.ac	2006-11-10 01:52:10.123157172 +0100
@@ -17,7 +17,6 @@
 
 dnl These next few lines are needed only while libcroco is in our source tree.
 AC_PROG_CC
-AM_PROG_CC_C_O
 if test "$GCC" = "yes"; then
   # Enable some warnings from gcc.
 
@@ -36,6 +35,7 @@
 AM_PROG_CC_STDC
 AM_PROG_AS
 AC_HEADER_STDC
+AM_PROG_CC_C_O
 
 dnl Honor aclocal flags
 ACLOCAL="$ACLOCAL $ACLOCAL_FLAGS"
