diff -urN inkscape-0.48.4.org/configure.ac inkscape-0.48.4/configure.ac
--- inkscape-0.48.4.org/configure.ac	2012-12-15 17:50:19.000000000 +0100
+++ inkscape-0.48.4/configure.ac	2013-04-18 08:23:52.486037399 +0200
@@ -21,12 +21,12 @@
   export CC CXX
 ])
 
-AM_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADERS(config.h)
 
 AC_LANG(C++)
 AC_ISC_POSIX
+AC_PROG_CC
 AC_PROG_CXX
-AM_PROG_CC_STDC
 AM_PROG_AS
 AC_PROG_RANLIB
 AC_PROG_INTLTOOL(0.22)
@@ -37,7 +37,6 @@
 m4_ifdef([AM_SILENT_RULES],[AM_SILENT_RULES([yes])])
 
 dnl These next few lines are needed only while libcroco is in our source tree.
-AC_PROG_CC
 AM_PROG_CC_C_O
 if test "$GCC" = "yes"; then
   # Enable some warnings from gcc.
