diff -burNp inkscape-0.46/src/widgets/sp-xmlview-attr-list.h inkscape-0.46-new/src/widgets/sp-xmlview-attr-list.h
--- inkscape-0.46/src/widgets/sp-xmlview-attr-list.h	2008-03-11 05:20:13.000000000 +0100
+++ inkscape-0.46-new/src/widgets/sp-xmlview-attr-list.h	2009-01-16 14:16:51.046936872 +0100
@@ -13,7 +13,7 @@
  */
 
 #include <stdio.h>
-#include <gtk/gtkclist.h>
+#include <gtk/gtk.h>
 #include "../xml/repr.h"
 
 #include <glib.h>
