diff -aurN inkscape-0.39.orig/configure.in inkscape-0.39/configure.in
--- inkscape-0.39.orig/configure.in	2004-06-05 13:45:32.000000000 +0200
+++ inkscape-0.39/configure.in	2004-06-06 13:40:17.192892064 +0200
@@ -54,7 +54,7 @@
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE",[Translation domain used])
 dnl Add the languages which your application supports here.
-ALL_LINGUAS="am az be ca cs da de el es es_MX et fr ga gl hu it ja nl no ru sk sl sr sr@Latn sv tr uk pl pt_BR pt zh_CN"
+ALL_LINGUAS="am az be ca cs da de el es es_MX et fr ga gl hu it ja nb nl ru sk sl sr sr@Latn sv tr uk pl pt_BR pt zh_CN"
 AM_GLIB_GNU_GETTEXT
 
 AC_PATH_PROG(PKG_CONFIG, pkg-config, no)
