--- inkscape-0.92.1/configure.ac.orig	2017-06-28 19:16:17.655179973 +0200
+++ inkscape-0.92.1/configure.ac	2017-06-28 20:40:42.491788807 +0200
@@ -723,8 +723,8 @@
 if test "x$enable_gtk3" = "xyes"; then
 
 	ink_spell_pkg=
-	if pkg-config --exists gtkspell-3.0; then
-		ink_spell_pkg=gtkspell-3.0
+	if pkg-config --exists gtkspell3-3.0; then
+		ink_spell_pkg=gtkspell3-3.0
 		AC_DEFINE(WITH_GTKSPELL, 1, [enable gtk spelling widget])
 	fi
 
